src/widget/weffectparameternamebase.cpp

```diff
@@ -9,12 +9,17 @@
 
 namespace {
 const QString kMimeTextDelimiter = QStringLiteral("\n");
+// for rounding the value display to N decimals
+const int kValDecimals = pow(10, 2);
 } // anonymous namespace
 
 WEffectParameterNameBase::WEffectParameterNameBase(
         QWidget* pParent, EffectsManager* pEffectsManager)
         : WLabel(pParent), m_pEffectsManager(pEffectsManager) {
     setAcceptDrops(true);
+    m_valDisplayTimer.setSingleShot(true);
+    m_valDisplayTimer.setInterval(800);
+    m_valDisplayTimer.callOnTimeout(this, &WEffectParameterNameBase::parameterUpdated);
```

Comment:
I think it is required to split parameterUpdated() to 
parameterSlotUpdated() an showParameterName() or such.

Reply:
Maybe the connect part needs to be moved to setEffectParameterSlot() ... ?

Reply:
oh, I totally overlooked that when reviving this old branch.
`callOnTimeout(this, [this]() {setText(m_text);})` is sufficient.